src/tidecli/models/task_data.py

```diff
@@ -108,6 +108,12 @@ class TaskData(BaseModel):
     max_points: float | None = None
     """Maximum points for the task"""
 
+    deadline: str | None = None
+    """ISO8601 date string for the task deadline. Server time is UTC+0, so users of this property will need to account for the local offset."""
+
+    answer_limit: int | None = None
+    """ Maximum number of answers that the user can save in the task."""
```

Comment:
Extra space in the beginning.

the meaning of "answer that the use can save in the task" is slightly unclear to me. What does it mean to save an answer in a task?

 ```suggestion
    """Maximum number of submission attempts for the task."""
```